pkg/kdump/kdump-view.jsx

```diff
@@ -566,15 +529,37 @@ ${enableCrashKernel}
             );
         }
 
-        let kdumpSwitch = (<Switch isChecked={!!serviceRunning}
-                              onChange={this.props.onSetServiceState}
-                              aria-label={_("kdump status")}
-                              isDisabled={this.props.stateChanging || !this.props.kdumpCmdlineEnabled} />);
+        let kdumpSwitch;
+        let kdumpSwitchHelper;
         if (!this.props.kdumpCmdlineEnabled) {
-            kdumpSwitch = (
-                <Tooltip content={_("crashkernel not configured in the kernel command line")} position={TooltipPosition.right}>
-                    {kdumpSwitch}
-                </Tooltip>);
+            kdumpSwitchHelper = _("Currently not supported");
+        } else {
+            kdumpSwitch = (<Switch isChecked={!!serviceRunning}
+                onChange={this.props.onSetServiceState}
+                aria-label={_("kdump status")}
+                isDisabled={this.props.stateChanging} />);
+            kdumpSwitchHelper = serviceRunning ? _("Enabled") : _("Disabled");
+        }
+
+        let alertMessage;
+        let alertDetail;
+        if (!this.props.stateChanging && this.props.kdumpStatus && this.props.kdumpStatus.installed !== undefined) {
+            if (this.props.kdumpStatus.installed) {
+                if (this.props.reservedMemory == 0) {
+                    alertMessage = _("Parameter crashkernel not configured in the kernel command line");
```

Comment:
```suggestion
                    alertMessage = _('Kernel did not boot with the "crashkernel" setting');
```

Reply:
This could also be backticks.

```suggestion
                    alertMessage = _("Kernel did not boot with the `crashkernel` setting");
```

However, I'm not sure how it would look with a `<pre>` in the title.